OrbitGl/App.cpp

```diff
@@ -805,15 +805,27 @@ void OrbitApp::LoadModules(int32_t process_id,
 //-----------------------------------------------------------------------------
 void OrbitApp::LoadModulesFromPreset(const std::shared_ptr<Process>& process,
                                      const std::shared_ptr<Preset>& preset) {
-  std::vector<std::shared_ptr<Module>> modules;
+  std::vector<std::shared_ptr<Module>> modules_to_load;
   for (const auto& pair : preset->m_Modules) {
     const std::string& module_path = pair.first;
     const auto& module = process->GetModuleFromPath(module_path);
-    if (module != nullptr && !module->IsLoaded()) {
-      modules.emplace_back(std::move(module));
+    if (module == nullptr) {
+      SendErrorToUi(
+          "Session loading incomplete",
```

Comment:
Session -> Preset?

Reply:
Yes, of course. I changed it here and where I copied it from